packages/tpetra/core/cmake/ExplicitInstantiationSupport.cmake

```diff
@@ -168,6 +173,27 @@ TRIBITS_ETI_GENERATE_MACROS(
     "TPETRA_INSTANTIATE_CONVERT_NOGPU(SOUT,SIN,LO,GO,N)"  TPETRA_ETIMACRO_CONVERT_NOGPU
     "TPETRA_INSTANTIATE_CONVERT_NOGPU_SSL(SOUT,SIN,LO)"    TPETRA_ETIMACRO_CONVERT_NOGPU_SSL)
 
+#SET(${PACKAGE_NAME}_SCALAR_INT_ETI_FIELDS "LO|GO|N")
+#TRIBITS_ETI_TYPE_EXPANSION(${PACKAGE_NAME}_SCALAR_INT_ETI_LIBRARYSET 
+#  "LO=${${PACKAGE_NAME}_ETI_LORDS}" 
+#  "GO=${${PACKAGE_NAME}_ETI_GORDS}"
+#  "N=${${PACKAGE_NAME}_ETI_NODES}")
+
+
+# For Mutlivector and friends
+TRIBITS_ETI_GENERATE_MACROS(
+    "${${PACKAGE_NAME}_ETI_FIELDS}" "${${PACKAGE_NAME}_ETI_LIBRARYSET}" 
+    "${${PACKAGE_NAME}_ETI_EXCLUDE_SET_GORD_NOT_INT}"
+    list_of_manglings eti_typedefs
+    "TPETRA_INSTANTIATE_GLGN_NO_INT(GO,LO,GO,N)"           TPETRA_ETIMACRO_GLGN_NO_INT)
+
+TRIBITS_ETI_GENERATE_MACROS(
+    "${${PACKAGE_NAME}_ETI_FIELDS}" "${${PACKAGE_NAME}_ETI_LIBRARYSET}" 
+    "${${PACKAGE_NAME}_ETI_EXCLUDE_SET}"
+    list_of_manglings eti_typedefs
+    "TPETRA_INSTANTIATE_LLGN(LO,LO,GO,N)"           TPETRA_ETIMACRO_LLGN)
```

Comment:
Per e-mail discussion, this is a hack that relies on `LO=int`, but I'm OK with that.  We will get rid of GO as a template parameter soon enough, and that will require rewriting all the ETI logic anyway, so we can tolerate some hacks for now.